corehq/apps/reports/static/reports/js/bootstrap3/case_details.js

```diff
@@ -72,20 +72,13 @@ hqDefine("reports/js/bootstrap3/case_details", [
 
         self.data_loading = ko.observable(false);
 
-        self.getParameterByName = function (name, url) {
-            if (!url) {
-                url = window.location.href;
+        self.getFormIdFromUrl = function () {
+            var regex = new RegExp("[?&]form_id(=([^&#]*)|&|#|$)"),
+                results = regex.exec(window.location.hash);
+            if (results && results[2]) {
+                return decodeURIComponent(results[2]);
             }
-            name = name.replace(/[[\]]/g, "\\$&");
-            var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)"),
-                results = regex.exec(url);
-            if (!results) {
-                return null;
-            }
-            if (!results[2]) {
-                return '';
-            }
-            return decodeURIComponent(results[2].replace(/\+/g, " "));
```

Comment:
There shouldn't be any `+`s in a form ID.  Probably don't even need the `decodeURIComponent` call, but I left that in just in case weird input would impact something.